pocs/observatory.py

```diff
@@ -493,6 +499,96 @@ def close_dome(self):
             self.logger.info('Closed dome')
         return self.dome.close()
 
+    def calibrate_mount(self, target=None, exp_time=30):
+        """Calibrates the mount to a target.
```

Comment:
What does calibration achieve? Reading the steps I don't get a clue of what the end result is.

Reply:
I'll fill out the docs a little more, thanks.